tests/boardfarm_plugins/boardfarm_prplmesh/testsuites.cfg

```diff
@@ -7,3 +7,4 @@
 InitialApConfig
 ApConfigRenew
 ClientAssociationLinkMetrics
+LinkMetricQuery
```

Comment:
**Note**: Github gives a warning about missing new line.

Reply:
Also please verify when you commit something that there are no spurious changes.